wdl-engine/src/eval/v1/task.rs

```diff
@@ -1117,7 +1117,7 @@ impl TaskEvaluator {
 
         // Localize the inputs
         self.backend
-            .localize_inputs(&self.downloader, &mut inputs)
+            .localize_inputs(Arc::new(self.downloader.clone()), &mut inputs)
```

Comment:
You actually aren't going to need this `Arc` I don't believe. The reason is that the `HttpDownloader` can be cheaply cloned, as it uses a bunch of `Arc`s internally. We should rearchitect this to not use the outer `Arc` if possible.

Reply:
Replaced `dyn Downloader` with `HttpDownloader`  but I wanted to confirm if it's fine in case we need to extend the downloader later for mocks?

Reply:
That's a good thought. For now, I think it's fine to keep it as is. If we do need to use mocks in the future, I'd probably reach for using generics rather than `&dyn Downloader` or `Arc<dyn Downloader>`.